poppy/manager/default/services.py

```diff
@@ -74,6 +79,15 @@ def determine_sleep_times(self):
         return backoff
 
     def _get_provider_details(self, project_id, service_id):
+        """Return Provider details for given project and service ids.
+
+        :param unicode project_id: The project id
+        :param unicode service_id: The service id
+        :return: Provider details for the given project and service ids
+        :rtype: Dict[str, poppy.model.helpers.provider_details.ProviderDetail]
```

Comment:
`Dict` (with capital 'D') is inconsistent. Also, I don't believe pycharm handles it correctly when displaying 'quick-help'.